src/types/sections.d.ts

```diff
@@ -104,3 +104,48 @@ export interface HeaderSectionProps {
   handleItemClick?: (item: Breadcrumb) => void;
 }
 export class HeaderSection extends Component<HeaderSectionProps> {}
+
+export interface FullWidthImageSliderSectionSlide {
+  /**
+   * A resolution map which provides the width / height and url for each resolution
+   *
+   * The Section will automatically make sure, that the correct image is referenced
+   */
+  image: Record<
+    string,
+    {
+      url: string;
+      width: number;
+      height: number;
+    }
+  >;
+  /**
+   * The title of the slide that will be animated in.
+   *
+   * Note: This can contain RichText
+   */
+  title: string;
+  /**
+   * The description that will be animated. Makre sure that you do not use too much text.
```

Comment:
Little typo in "Makre":
```suggestion
   * The description that will be animated. Make sure that you do not use too much text.
```